<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminframe</class>
 <widget class="QDialog" name="adminframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#adminframe{background:url(:/new/prefix1/adminframe.png);}</string>
  </property>
  <widget class="QPushButton" name="adduserButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>添加帐号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbookButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>添加书籍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>帐号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccount">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>13020031029</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>201</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>帐号查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newpassEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>101</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>帐号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultAccount">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultKind">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newpassEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>删除帐号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchbook">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 85, 0)</string>
   </property>
   <property name="text">
    <string>请输入索书号进行操作：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="result01">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="result02">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLabel" name="result03">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>出版社：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="borrowButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>借阅</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>归还</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBook">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="bottom1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>输入用户：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bottom2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>590</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottom3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bottom4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="result1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="result2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="result3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="warn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="result04">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>书号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="result4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 103, 0)</string>
   </property>
   <property name="text">
    <string>温馨提示：查询操作后有借阅、归还、删除图书功能</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background:#fe7023;
color:#ffffff;
font: 10pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>新开查询</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>661</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>newpassEdit</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>adduserButton</zorder>
  <zorder>exitButton</zorder>
  <zorder>addbookButton</zorder>
  <zorder>changeButton</zorder>
  <zorder>label</zorder>
  <zorder>labelAccount</zorder>
  <zorder>searchEdit</zorder>
  <zorder>searchButton</zorder>
  <zorder>label_2</zorder>
  <zorder>resultAccount</zorder>
  <zorder>label_3</zorder>
  <zorder>resultKind</zorder>
  <zorder>newpassEdit_2</zorder>
  <zorder>deleteButton</zorder>
  <zorder>searchbook</zorder>
  <zorder>label_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>result01</zorder>
  <zorder>result02</zorder>
  <zorder>result03</zorder>
  <zorder>borrowButton</zorder>
  <zorder>returnButton</zorder>
  <zorder>deleteBook</zorder>
  <zorder>bottom1</zorder>
  <zorder>bottom2</zorder>
  <zorder>bottom3</zorder>
  <zorder>bottom4</zorder>
  <zorder>result1</zorder>
  <zorder>result2</zorder>
  <zorder>result3</zorder>
  <zorder>warn</zorder>
  <zorder>result04</zorder>
  <zorder>result4</zorder>
  <zorder>label_5</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>changeButton_2</zorder>
  <zorder>line_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
